joomla/joomla-cms
# Update Install from Web plugin text

Pull Request for Issue #11108 

simple PR to correct the language string to remove the word obsolete as identified by @PhilETaylor 
#### Testing Instructions

Its quite tricky to test. To display the message do the following
1. Install the Install from web plugin
2. Edit /plugins/installer/webinstaller/webinstaller.xml to set the < version > tag to 0.9
3. Go to Extensions install and you should now see the error message
4. Apply the patch and see the changed message
